Bottom: 8e9c83c00de139d5ea11e005aa85712cfdc9415d
Top:    34409584bb26fcb7aac4293d3f6a62e2fd99ee49
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 18:24:00 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
